tasks/hyrax-metadata-updates/index.js

```diff
@@ -91,7 +91,8 @@ async function getEntryTitle(config, metadata, isUmmG) {
   const cmrInstance = new CMR({
     provider: config.cmr.provider,
     username: config.cmr.username,
-    password: config.cmr.passwordSecretName
+    password: config.cmr.passwordSecretName,
+    clientId: config.cmr.clientId
```

Comment:
What was setting the `clientId` here before? Or is it a non-required request header?

Reply:
That's a good question and it took me a while to remember, but I have. So, nothing. But the part that hollers at you is in the token code and we were just using this for search which wasn't using a token. I added the token to the search code since it would have to look for private collections, then got the error and added it.